pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -886,10 +886,10 @@ cdef int64_t parse_iso_format_string(str ts) except? -1:
                     neg = 1
             elif c == "+":
                 pass
-            elif c in ["W", "D", "H", "M"]:
-                if c in ["H", "M"] and len(number) > 2:
+            elif c in ["W", "D", "H", "ME"]:
+                if c in ["H", "ME"] and len(number) > 2:
                     raise ValueError(err_msg)
-                if c == "M":
+                if c == "ME":
```

Comment:
within this function (`parse_iso_format_string`), `'M'` actually means `minute`, it's a specific format: https://en.wikipedia.org/wiki/ISO_8601#Durations

so, this particular function shouldn't change

Reply:
Thank you, I corrected these lines.